src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -102,26 +102,24 @@ private void LogAnalyzerResult(IAnalyzerResult analyzerResult)
 
         private IAnalyzerResult SelectAnalyzerResult(IAnalyzerResults analyzerResults, string targetFramework)
         {
+            var firstAnalyzerResult = analyzerResults?.First(e => e.TargetFramework is not null);
             if (targetFramework == null)
```

Comment:
```suggestion
            if (targetFramework is null)
```

Reply:
My bad. You've made this suggestion before and I didn't take care of it. 

On another note, good heavens you are fast.

Reply:
I can also be very slow :)